lib/reporters/spec.js

```diff
@@ -67,20 +67,35 @@ function Spec(runner, options) {
   });
 
   runner.on(EVENT_TEST_PASS, function (test) {
+    function logRetries() {
+      if (test._retries > 0) {
+        var retryFmt = indent() + color('bright yellow', '  %s');
+        var retryMessage =
+          test._currentRetry > 0
+            ? `(Succeeded after ${test._currentRetry} / ${test._retries} retries)`
+            : '';
+
+        if (retryMessage) {
+          Base.consoleLog(retryFmt, retryMessage);
+        }
+      }
+    }
     var fmt;
     if (test.speed === 'fast') {
       fmt =
         indent() +
         color('checkmark', '  ' + Base.symbols.ok) +
         color('pass', ' %s');
       Base.consoleLog(fmt, test.title);
+      logRetries();
     } else {
       fmt =
         indent() +
         color('checkmark', '  ' + Base.symbols.ok) +
         color('pass', ' %s') +
         color(test.speed, ' (%dms)');
       Base.consoleLog(fmt, test.title, test.duration);
+      logRetries();
     }
```

Comment:
[Refactor] Both the `if` and the `else` blocks end with `logRetries()`. So the first can be removed:

```suggestion
    }
    
    logRetries();
```